### DISCUSSION TITLE
Miscellaneous Questions [laptops, wifi, health-changes,..]

### DISCUSSION THREAD

**Discussion by katreparitosh** [snigi-gupta/AttendingUniversityAtBuffalo#17] [General]
Hello,

First of all, a big thanks to you and all the contributors for open-sourcing so much information! 

After reading through the discussion threads, I had a few miscellaneous queries:   

1. What laptop would you suggest for a graduate student (CS, IE, any course..)? Would a good MacBook Air /  Pro be enough for all tasks or would you recommend a good windows machine with enough specs?
2. Which WiFi providers in South Campus are good and reliable? Any suggestions?
3. As mentioned by you earlier, the weather in Buffalo is different than what we experience in India. Hence, any health-related problems people face due to harsh variance in weather. Example: skin problems, changes in hair due to water quality, etc.?

Regards,
Paritosh

**Comment by snigi-gupta** [comment]
@katreparitosh,

I had a dell inspiron 15 700 with 1TB storage and 8GB RAM. It worked superb for me. My friend had a Macbook air and that too worked for him. Another friend of mine had a Razor laptop with Linux OS (booted) and that too worked for him. 
You can buy any laptop that is convenient for you and your budget. Just make sure the laptop is not too heavy as carrying it all the time can be painful to your back. For any machine-related operations, we all used google colab since gpu could be used. Or I used paperspace. So for high-intensity computations, gpus are available out there.

For wifi, you can go with spectrum or Verizon. I have used both. I switched from spectrum to Verizon because I was having too frequent network drops with spectrum, hence I decided to switch. Verizon is working great for now, for me.

No problems were faced due to the cold. We get accustomed. If you have a particular health condition already, or you catch cold-cough or flu frequently, make sure to get your meds in advance. No such skin problems. I did have some hair fall due to the fluorine water here in the initial days but once accustomed, it was fine. The rest of my hair fall was due to stress 🤣

**Reply by katreparitosh** [reply]
@snigi-gupta, 

Haha! Thanks for the info :)
